tools/remote_build/linux.bazelrc

```diff
@@ -13,42 +13,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# bazelrc with Foundry setting common to both manual run and runs started by Kokoro
-# see https://github.com/bazelbuild/bazel-toolchains/tree/master/bazelrc
-# for examples and more documentation
+# bazelrc file for running gRPC tests with RBE (on Linux)
 
-startup --host_jvm_args=-Dbazel.DigestFunction=SHA256
+import %workspace%/tools/remote_build/include/rbe_remote_execution.bazelrc
 
+# Next section is linux-specific RBE configuration
 build --crosstool_top=@rbe_default//cc:toolchain
 build --extra_toolchains=@rbe_default//config:cc-toolchain
 # Use custom execution platforms defined in third_party/toolchains
 build --extra_execution_platforms=@rbe_default//config:platform
 build --host_platform=@rbe_default//config:platform
 build --platforms=@rbe_default//config:platform
 
-build --spawn_strategy=remote
-build --strategy=Javac=remote
-build --strategy=Closure=remote
-build --genrule_strategy=remote
-build --remote_timeout=7200  # very large value to avoid problems like https://github.com/grpc/grpc/issues/20777
-
-build --remote_instance_name=projects/grpc-testing/instances/default_instance
-
-build --verbose_failures=true
-
-build --experimental_strict_action_env=true
-build --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
-
-# don't use port server
-build --define GRPC_PORT_ISOLATED_RUNTIME=1
-# without verbose gRPC logs the test outputs are not very useful
-test --test_env=GRPC_VERBOSITY=debug
-
 # we assume the default bazel RBE build is on linux,
 # so filter out stuff that should not be built or run there.
 build --test_tag_filters=-no_linux
 build --build_tag_filters=-no_linux
 
+import %workspace%/tools/remote_build/include/test_config_common.bazelrc
+
+build --jobs=100
```

Comment:
Do we want to have concurrency of 100 here? Does this mean 100 containers in RBE (different than local runs)? If we only have 1 machine, will 100 jobs slow down the compilation?

Reply:
Yes, that's 100 parallel jobs when running on RBE cluster, when started as a "manual" RBE run from developer's workstation. When RBE build is started from CI, we use 200 parallel jobs. 
The idea is that sometimes folks run jobs with lots of repeated tests (e.g. `--runs_per_test 10000` when deflaking a test) and we don't want them to negatively impact the throughput of the CI, so they get let lower "quota" than the CI.

Reply:
Thanks for the info. I thought we only have CPUs in magnitude of hundreds, I was wrong. 100 jobs should be plenty for normal developments, and it should not impact other jobs in normal cases.